docs/en/data_types/datetime.md

```diff
@@ -1,15 +1,58 @@
 # DateTime {#data_type-datetime}
 
-Date with time. Stored in four bytes as a Unix timestamp (unsigned). Allows storing values in the same range as for the Date type. The minimal value is output as 0000-00-00 00:00:00.
-The time is stored with accuracy up to one second (without leap seconds).
+Data structure storing Unix timestamp (`UInt32`) and a time zone.
 
-## Time Zones
+Syntax:
 
-The date with time is converted from text (divided into component parts) to binary and back, using the system's time zone at the time the client or server starts. In text format, information about daylight savings is lost.
+```sql
+DateTime([timezone])
+```
 
-By default, the client switches to the timezone of the server when it connects. You can change this behavior by enabling the client command-line option `--use_client_time_zone`.
+Range: [1970-01-01 00:00:00, 2105-12-31 23:59:59].
```

Comment:
Wrong. It depends on a timezone. For example, 1970-01-01 00:00:01 is not supported in Moscow, because it is before unix epoch.